### DISCUSSION TITLE
Clarity on Repository's and cost

### DISCUSSION THREAD

**Discussion by GamesAlchemist** [community/community#22098] [New to GitHub]
<p>Hi, I’m currently in the process of trying to set up version control for a games project working with a team remotely. I have had experience of using version control but not setting it up for use with a team using Unity.<br>
I’m struggling to understand how initially I would get a large project to another member of the team, (25 gigs without the project being organised)<br>
Would this need to be uploaded to the cloud initially then the team member download the project?<br>
Or by hosting the project locally on my PC can the team member clone the project from the repository created, pulling it directly from my PC?<br>
I’m just a little confused how the initial setup and transfer of files happens.<br>
Can this be done with a free account or will I have to pay for cloud storage through Github or use Microsoft Azure?</p>
<p>I don’t actually want to have the project left on the cloud if this is the case, Just for the initial transfer of files. I would prefer it to just work locally of my PC.</p>
<p>I’ve tried to seek an answer online but can’t really get any clarity on this for me to better understand.</p>
<p>If anybody could give me some advice it would be greatly appreciated.</p>
<p>Thanks in advance</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi :wave: welcome!</p>
<p>In order to host a repository containing 25GB, you would have to pay for Large File Storage.</p>
<p>You can read more here:</p>
<p><a href="https://help.github.com/github/managing-large-files/about-git-large-file-storage" class="onebox" target="_blank">https://help.github.com/github/managing-large-files/about-git-large-file-storage</a></p>
<p>If you would prefer to not pay, or you want to use a self-hosted solution, there is a list of various large file git implementations here:</p>
<aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/git-lfs/git-lfs/wiki/Implementations" target="_blank">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181068864-2b163cd2-b660-4a1f-8d27-9687cec6cbac.png" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/git-lfs/git-lfs/wiki/Implementations" target="_blank">git-lfs/git-lfs</a></h3>

<p>Git extension for versioning large files. Contribute to git-lfs/git-lfs development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you have questions about any of the non-GitHub solutions, I would strongly suggest opening a new thread in “How to use Git and GitHub” to get the right eyes on it.</p>
